### ISSUE TITLE
If video is opened as part of interact action, Insomniac crashes

### ISSUE THREAD

**Issue by Ardy000** [alexal1/Insomniac#239]
```
[01/05 22:57:25] Double click!
WARNING:root:-32001 Jsonrpc error: <android.support.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.FrameLayout, RESOURCE_ID_REGEX=com.instagram.android:id/zoomable_view_container|com.instagram.android:id/carousel_image], method: objInfo, retrying in 0.5 seconds...
WARNING:root:-32001 Jsonrpc error: <android.support.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.FrameLayout, RESOURCE_ID_REGEX=com.instagram.android:id/zoomable_view_container|com.instagram.android:id/carousel_image], method: objInfo, retrying in 0.6 seconds...
[01/05 22:57:27] Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/device_facade.py", line 474, in get_bounds
    return self.viewV2.info['bounds']
  File "<decorator-gen-4>", line 2, in info
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/retry/api.py", line 73, in retry_decorator
    return __retry_internal(partial(f, *args, **kwargs), exceptions, tries, delay, max_delay, backoff, jitter,
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/uiautomator2/_selector.py", line 130, in info
    return self.jsonrpc.objInfo(self.selector)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/uiautomator2/__init__.py", line 435, in __call__
    return self.server._jsonrpc_retry_call(self.method, params,
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/uiautomator2/__init__.py", line 442, in _jsonrpc_retry_call
    return self._jsonrpc_call(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/uiautomator2/__init__.py", line 514, in _jsonrpc_call
    raise err
uiautomator2.exceptions.UiObjectNotFoundError: -32001 Jsonrpc error: <android.support.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.FrameLayout, RESOURCE_ID_REGEX=com.instagram.android:id/zoomable_view_container|com.instagram.android:id/carousel_image], method: objInfo

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/safely_runner.py", line 14, in wrapper
    func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/action_runners/interact/__init__.py", line 162, in job
    handle_hashtag(device_wrapper.get(),
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/action_runners/interact/action_handle_hashtag.py", line 211, in handle_hashtag
    extract_hashtag_profiles_and_interact(device,
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/action_runners/interact/action_handle_hashtag.py", line 270, in extract_hashtag_profiles_and_interact
    should_continue_using_source = iterate_over_likers(device, iteration_callback, pre_conditions)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/actions_impl.py", line 204, in iterate_over_likers
    to_continue = iteration_callback(username, user_name_view)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/action_runners/interact/action_handle_hashtag.py", line 171, in interact_with_profile
    is_liked, is_followed, is_watch, is_commented = interaction(username=liker_username, interaction_strategy=interaction_strategy)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/actions_impl.py", line 312, in interact_with_user
    do_like_actions()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/actions_impl.py", line 286, in do_like_actions
    if not _open_photo_and_like_and_comment(device, row, column,
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/actions_impl.py", line 363, in _open_photo_and_like_and_comment
    post_view.double_click()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/device_facade.py", line 393, in double_click
    self._double_click_v2(padding)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/device_facade.py", line 561, in _double_click_v2
    visible_bounds = self.get_bounds()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/device_facade.py", line 476, in get_bounds
    raise DeviceFacade.JsonRpcError(e)
insomniac.device_facade.DeviceFacade.JsonRpcError: -32001 Jsonrpc error: <android.support.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.FrameLayout, RESOURCE_ID_REGEX=com.instagram.android:id/zoomable_view_container|com.instagram.android:id/carousel_image], method: objInfo
```

**Comment by Fadude** [comment]
@Ardy000  Can you send a zip-crash please?

**Comment by Ardy000** [comment]
Yeah I will next time it happens.

I was wrong to say it happened on video. I just watched it do video without issue. I'm not sure if it's only video. I've never seen it crash on pictures but it could have

**Comment by simonmannsfeld** [comment]
[Crash-2021-01-20-17-42-18.zip](https://github.com/alexal1/Insomniac/files/5844152/Crash-2021-01-20-17-42-18.zip)

I guess I had the same crash. It looks like insomniac is crashing on opening a carousel that starts with a video.

I created pull request #257 for this. It's still untested because I love the thill! Please triplecheck - I don't know what I am doing. lol 😆 Please merge and close if it works.

**Comment by simonmannsfeld** [comment]
Carousels that start with a video uses the resource-id `com.instagram.android:id/carousel_video_image`. Just waiting for someone who can fix. 😄

**Comment by alexal1** [comment]
Fixed in `v3.7.0`
